notebooks/official/automl/automl-tabular-classification.ipynb

```diff
@@ -51,7 +51,7 @@
         "id": "tvgnzT1CKxrO"
```

Comment:
<p>I'll fix this Vertex AI Vertex AI business...</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/123/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>